Extensions/BBText/bbtextruntimeobject.js

```diff
@@ -0,0 +1,200 @@
+/**
+ * Displays a rich text using BBCode markup (allowing to set parts of the text as bold, italic, use different colors and shadows).
+ * @memberof gdjs
+ * @class BBTextRuntimeObject
+ * @extends RuntimeObject
+ */
+gdjs.BBTextRuntimeObject = function(runtimeScene, objectData) {
+  gdjs.RuntimeObject.call(this, runtimeScene, objectData);
+
+  /** @type number */
+  this._opacity = objectData.content.opacity;
+  /** @type boolean */
+  this._visible = objectData.content.visible;
+  /** @type string */
+  this._text = objectData.content.text;
+  /** @type string */
+  this._color = objectData.content.color;
+  /** @type string */
+  this._fontFamily = objectData.content.fontFamily;
+  /** @type number */
```

Comment:
Isn't fontSize a string? Things are a bit unclear, I would recommend setting this as a number everywhere.

Reply:
(everywhere = in JsExtension.js too)

Reply:
that one is a bit strange :) to the pixi multistyle object it needs to be a string (+px), but to GD it is a number